runtime/src/main.rs

```diff
@@ -148,6 +149,17 @@ fn autoconfigure_actors(
     }
 }
 
+/// Returns a capability provider's capability ID.
+fn capability_id(instance: &impl CapabilityProvider) -> anyhow::Result<String> {
+    let bytes: [u8; 0] = [];
+    let result = instance
+        .handle_call(SYSTEM_ACTOR, OP_GET_CAPABILITY_DESCRIPTOR, &bytes[..])
```

Comment:
There's a convenient shorthand where you don't need to create an empty array to get an empty slice... just passing `&[]` will work on `handle_call`.

Reply:
Thanks. I though there must be a better way.